nilearn/plotting/img_plotting.py

```diff
@@ -232,11 +261,13 @@ def _crop_colorbar(cbar, cbar_vmin, cbar_vmax):
         return
     cbar_tick_locs = cbar.locator.locs
     if cbar_vmax is None:
-        cbar_vmax = cbar_tick_locs.max()
+        cbar_vmax = cbar.norm.vmax
```

Comment:
to check: where did `_crop_colorbar` go?

Reply:
IIRC we got rid of it in #3188  [see here](https://github.com/nilearn/nilearn/pull/3188/files#diff-94cfb3db2e8d3016f3b6850b4943c90c6d0dbce93997772f3ce67fff57276187)

but you may need to check if it affects this PR somehow!

Reply:
thanks: avoids doing quite a bit of code archeology!!